frontend/src/app/components/containers/complaints/complaint-filter.tsx

```diff
@@ -22,6 +23,8 @@ import { ComplaintFilterPayload, updateFilter } from "../../../store/reducers/co
 import Option from "../../../types/app/option";
 import { getUserAgency } from "../../../service/user-service";
 import { listActiveFilters } from "../../../store/reducers/app";
+import UserService from "../../../service/user-service";
```

Comment:
As per the sonar cloud comment, the previous import of { getUserAgency } is no longer required.